src/material/sidenav/sidenav.md

```diff
@@ -209,6 +209,11 @@ Similarly, the `<mat-sidenav-content>` should be given a role based on what it c
 represents the primary content of the page, it may make sense to mark it `role="main"`. If no more
 specific role makes sense, `role="region"` is again a good fallback.
 
+#### Focus management
+The sidenav captures focus automatically by default, unless it's in the side mode. Focus Capturing can be enabled or disabled explicitly, using `autoFocus` input
```

Comment:
Sorry, a couple of last nits and then it's good to go:
- "Focus capturing" without the extra capitalization.
- "using `autoFocus` input" -> "using the `autoFocus` input"
- It needs a full stop at the end of the sentence.

Reply:
yeah no prob, I think that's correct right?
> Focus capturing can be enabled or disabled explicitly, using the `autoFocus` input.

Reply:
Yes.